### PR TITLE
Enable SCSS files in the css

### LINKED ISSUE #243: Support to Sass
Hi,

When we include a `scss` file into the `pagedown:html_paged`, I get the following error:

```
Error: "print.scss" is not a valid built-in CSS filename. Use `pagedown:::list_css()` to view all built-in CSS filenames.
```

The frontmatter relevant part is next:

```yaml
output: 
    pagedown::html_paged:
      css:
        - print.scss
```

Maybe removing the filter for "valid" files may solve this issue.

### PR DESCRIPTION
**Pull request by felipecrp**
This PR fix #243.

I changed the regexp to enable `css` and `scss` files.

Best regards,
Felipe

### CODE REVIEW

**Comment by felipecrp** [comment]
@yihui Apparently, the error is related to the build process instead of the PR change itself. Can you help me fix it?

File: R/paged.R [hunk 1]
```diff
@@ -148,7 +148,7 @@ html_format = function(
     }
   }
 
-  css2 = grep('[.]css$', css, value = TRUE, invert = TRUE)
+  css2 = grep('[.]s?css$', css, value = TRUE, invert = TRUE)
```
Comment (cderv) [inline R/paged.R:151]: To be complete, the regex would be 
```suggestion
  css2 = grep('[.](?:sa|sc|c)ss$', css, value = TRUE, invert = TRUE)
```

**Review by cderv — CHANGES_REQUESTED** [review]
Thanks. 

Following support in `rmarkdown::html_document()` we indeed did not bring support up to **pagedown**. I think we can do that. Thanks for catching that!

In addition of the regex, it would require to 

1. update the `css` argument description on the `html_paged` documentation in the roxygen comment
2. Add a NEWS bullet
3. Add a mention of SASS support in the main documentation (`inst/examples/index.Rmd`) under `## The CSS overriding mechanism` part.

**Comment by yihui** [comment]
@felipecrp We can ignore the Travis error in this case. Thanks!

File: R/paged.R (hunk 1 above)
Comment (felipecrp) [reply R/paged.R:151]: Ok. I will test the changes and update the docs.

**Comment by felipecrp** [comment]
@cderv and @yihui ,

Thank you for the support. 

I changed the regexp using the @cderv suggestion and tested the new release with `.scss` and `.sass` files. I also updated the roxygen doc, the documentation, and the NEWS.

I think this PR is ready for delivery.

Best regards,
Felipe

**Review by cderv — CHANGES_REQUESTED** [review]
Thanks ! 

We still need to increase the min dependency of **rmarkdown** to 2.8 because of this feature. 
Currently it is 1.16
https://github.com/rstudio/pagedown/blob/bdc6a9285bc60e8b93756b30e919c10923088c72/DESCRIPTION#L23

This is the good practice, right @yihui ? 
I don't see the harm in doing that, all the more because it is always best to use the latest **rmarkdown** available.

File: NEWS.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 
 - In `chrome_print()`, set a timeout when R tries to detect headless Chrome.
 
+- Added support for Sass files.
```
Comment (cderv) [inline NEWS.md:11]:
```suggestion
- Added support for Sass files with `.scss` and `.sass` extension (thanks, @felipecrp, #244).
```

We usually thanks contributor in the NEWS item - so thank you ! :)

Comment (felipecrp) [reply NEWS.md]: You´re welcome 😄

File: DESCRIPTION [hunk 3]
```diff
@@ -20,7 +20,7 @@ Description: Use the paged media properties in CSS and the JavaScript
   pages. Each page can have its page size, page numbers, margin boxes, and
   running headers, etc. Applications of this package include books, letters,
   reports, papers, business cards, resumes, and posters.
-Imports: rmarkdown (>= 1.16), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
+Imports: rmarkdown (>= 2.7), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```
Comment (cderv) [inline DESCRIPTION:23]:
```suggestion
Imports: rmarkdown (>= 2.8), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```

I thought we needed 2.8 (https://pkgs.rstudio.com/rmarkdown/news/index.html#rmarkdown-28). This is the one supporting the SASS files in `css`

**Review by cderv — CHANGES_REQUESTED** [review]

**Comment by yihui** [comment]
> This is the good practice, right @yihui ?

Yes, let's do that.

**Review by cderv — APPROVED** [review]
Thanks @felipecrp ! I'll merge this.

**Comment by felipecrp** [comment]
Thank you for the support.

**Pull request merged**